test/jest/summary-reporter.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Wrapper to export Jest's SummaryReporter for use in jest.config.js
+ *
+ * This is needed because Node v24 strictly enforces package exports,
+ * and we can't directly access '@jest/reporters/build/SummaryReporter'.
```

Comment:
I'm not following why the inability to import the file `build/SummaryReporter` is relevant here. It's exported as `SummaryReporter`, so we don't need to reference the file.

Why is this wrapper needed?

Reply:
I added the wrapper because I initially tried to use `@jest/reporters/build/SummaryReporter` (as shown in the example of jest-clean-console-reporter's [README](https://github.com/jevakallio/jest-clean-console-reporter?tab=readme-ov-file#configuration)) and that failed with Node v24's strict exports.

However, you're right, using [Jest's built-in 'summary' reporter](https://jestjs.io/docs/configuration#summary-reporter) works fine. It's now updated in this commit: https://github.com/MetaMask/metamask-extension/pull/38208/commits/ca26c784c7eec9548f770bcfde1a5ec8ee5289fe